pkg/network/config/config.go

```diff
@@ -141,8 +141,8 @@ func NewDefaultConfig() *Config {
 		UDPStreamTimeout:             defaultUDPStreamTimeoutSeconds * time.Second,
 		TCPConnTimeout:               2 * time.Minute,
 		TCPClosedTimeout:             time.Second,
-		MaxTrackedConnections:        65536,
-		ConntrackMaxStateSize:        65536,
+		MaxTrackedConnections:        131072,
+		ConntrackMaxStateSize:        131072,
```

Comment:
Do you have any estimates on what's the actual memory impact of this? I think for the eBPF map itself it's fine (should be around 20MB or so), but I guess there is footprint of the state map, conntrack among other things

Reply:
I don't see any pre-allocations using the maximum size of the userspace components. Let me do the math for the ebpf maps that use these values.

Reply:
`conn_stats` 48 byte key, 32 byte value = 5MB -> 10MB
`tcp_stats` 48 byte key, 16 byte value = 4MB -> 8MB
`port_bindings` 6 byte key, 1 byte value = 448KB -> 896KB
`udp_port_bindings` 6 byte key, 1 byte value = 448KB -> 896KB
`conntrack` (runtime compilation only) 48 byte key, 48 byte value = 6MB -> 12MB

So going from ~10MB or ~16MB -> ~20MB or ~32MB

Reply:
yeah, I'm not that concerned about the eBPF map size, but I'm wondering if is prudent to double this default param without knowing how much memory system-probe will use in _total_ (given the impact that this param has over all Go heap allocations). at the very minimum I think we should run this on a load test cluster to get a sense of that.
I personally would be more comfortable leaving the default value as it is and letting customers react to the log entry message.

Reply:
As I mentioned we don't pre-allocate, so memory usage from Golang maps would only go up if they were close to the limit anyways.

Reply:
I do understand that. what I'm suggesting is that we probably want to have a reasonable hard-limit on the resources system-probe can consume. you may say that we we should probably enforce that through other means (like in our helm-chart, or resource slices in our systemd daemon etc), but in any case we do need to have an idea of how much memory these new values can consume in order to have an informed discussion about this.
for example, it may be the case that if we have 131072 connections being tracked we'll OOM because the container limits on the helm chart are lower than the necessary memory.

Reply:
It looks like the increase in max connections increased the RSS by 80MiB.

Reply:
Is this needed anymore, now that the cache is an LRU?

Reply:
> It looks like the increase in max connections increased the RSS by 80MiB.

Apologies. This was incorrect. When hitting the new limit, the increase is around 200MiB.


> Is this needed anymore, now that the cache is an LRU?

The change in size for conntrack state is probably not needed, but `MaxTrackedConnections` increase is.

Reply:
@p-lambert Thoughts on the memory increase?

Reply:
humm, it does feel a considerable increase. I personally vote for keeping the current default values (given that they seem to be more than enough for the majority workloads), and let customers increase the limit as they see fit

Reply:
Now that we have the log messages, if we do have customers hitting the limits, it should be a bit more obvious to them. I'll revert the limit increase.